### DISCUSSION TITLE
ControlNet crashes for 100% because of old version of Models

### DISCUSSION THREAD

**Discussion by mylife4aiur5** [Mikubill/sd-webui-controlnet#82] [General]
![Screenshot 2023-02-15 at 11 40 34 PM](https://user-images.githubusercontent.com/121373688/219275511-f0412948-3b8b-473d-b68b-5af565d51129.jpg)
With the latest update, it seems v21 is required as whenever I use v15 model files, it crashes right away. I tested with ControlNet by using Preprocessor but without model and it works fine.

At this point, we really need v21 model files but I cant find anything. Any idea?

**Comment by Mikubill** [comment]
Seems like mps issue, maybe related: #68

**Reply by mylife4aiur5** [reply]
Huh, so no solutions for now?
